src/lib/table/row.ts

```diff
@@ -49,7 +49,7 @@ export class MatFooterRowDef extends _CdkFooterRowDef {}
 
 /**
  * Data row definition for the mat-table.
- * Captures the footer row's template and other footer properties such as the columns to display and
+ * Captures the data row's template and other properties such as the columns to display and
```

Comment:
I don't think this needs the "data" either, it can be just "Captures the row's template...".

Reply:
@crisbeto it can but I think it's more accurate to add "data" since first sentence states that it's "Data row definition for the mat-table."

Reply:
Also it's clearer when described this way that it only relates to normal data rows and not header or footer rows.